bigchaindb/tendermint/commands.py

```diff
@@ -37,36 +38,38 @@ def start():
         settings=bigchaindb.config['server'],
         log_config=bigchaindb.config['log'],
         bigchaindb_factory=BigchainDB)
-    p_webapi = mp.Process(name='webapi', target=app_server.run)
+    p_webapi = Process(name='webapi', target=app_server.run)
     p_webapi.start()
 
     # start message
     logger.info(BANNER.format(bigchaindb.config['server']['bind']))
 
-    if int(getenv('BIGCHAINDB_START_TENDERMINT', 1)):
+    if int(getenv('BIGCHAINDB_START_TENDERMINT', 0)):
```

Comment:
+1

Reply:
oh oh oh :santa: :santa: :santa: I forgot this. I think the default setting should be `0`, but I'll put it back to `1` because it doesn't belong here.